### PR TITLE
chore: [IAI-246] Changes pagopPa UAT API URI

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR changes the URI for pagoPa UAT APIs

### CODE REVIEW

File: .env.production [hunk 1]
```diff
@@ -5,7 +5,7 @@ API_URL_PREFIX='https://app-backend.io.italia.it'
 # pagoPA RESTful API
 PAGOPA_API_URL_PREFIX='https://wisp2.pagopa.gov.it/pp-restapi-CD'
 # pagoPA test/env RESTful API
-PAGOPA_API_URL_PREFIX_TEST='https://acardste.vaservices.eu:443/pp-restapi-CD'
+PAGOPA_API_URL_PREFIX_TEST='https://uat.wisp2.pagopa.gov.it/'
```
Comment (francescopersico) [inline .env.production:8]: I
```suggestion
PAGOPA_API_URL_PREFIX_TEST='https://uat.wisp2.pagopa.gov.it/pp-restapi-CD'
```

**Review by francescopersico — CHANGES_REQUESTED** [review]

File: .env.local [hunk 2]
```diff
@@ -5,7 +5,7 @@ API_URL_PREFIX='http://127.0.0.1:3000'
 # pagoPA RESTful API
 PAGOPA_API_URL_PREFIX='http://127.0.0.1:3000/wallet'
 # pagoPA test/env RESTful API
-PAGOPA_API_URL_PREFIX_TEST='https://acardste.vaservices.eu:443/pp-restapi-CD'
+PAGOPA_API_URL_PREFIX_TEST='https://uat.wisp2.pagopa.gov.it/'
```
Comment (francescopersico) [inline .env.local:8]:
```suggestion
PAGOPA_API_URL_PREFIX_TEST='https://uat.wisp2.pagopa.gov.it/pp-restapi-CD'
```

**Review by francescopersico — CHANGES_REQUESTED** [review]

File: .env.local [hunk 3]
```diff
@@ -5,7 +5,7 @@ API_URL_PREFIX='http://127.0.0.1:3000'
 # pagoPA RESTful API
 PAGOPA_API_URL_PREFIX='http://127.0.0.1:3000/wallet'
 # pagoPA test/env RESTful API
-PAGOPA_API_URL_PREFIX_TEST='https://acardste.vaservices.eu:443/pp-restapi-CD'
+PAGOPA_API_URL_PREFIX_TEST='https://uat.wisp2.pagopa.gov.it/pp-restapi-CD'
```
Comment (francescopersico) [inline .env.local:8]: Why we are using 12.0.0.1 for production and real URL for test?

Comment (CrisTofani) [reply .env.local:8]: on the local environment we don't have any differences between we may repeat `http://127.0.0.1:3000/wallet` btw nothing would change

**Review by emiliopavia — APPROVED** [review]

**Pull request merged**